server/src/main/java/org/spine3/server/entity/DefaultEntityStorageConverter.java

```diff
@@ -20,9 +20,11 @@
 
 package org.spine3.server.entity;
 
+import com.google.common.base.Optional;
 import com.google.protobuf.Any;
 import com.google.protobuf.FieldMask;
 import com.google.protobuf.Message;
+import org.spine3.server.entity.storagefield.StorageFieldsExtractor;
```

Comment:
I wouldn't call the package `storagefield`, but `storage` and then `.field` if it is really needed.